media-libs/panda3d/panda3d-9999.ebuild

```diff
@@ -0,0 +1,98 @@
+# Copyright 2008-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python{2_7,3_4,3_5} )
+
+inherit eutils multiprocessing python-single-r1
+
+if [[ ${PV} != 9999 ]]; then
+	MY_PV=v${PV}
+	KEYWORDS="~amd64 ~x86"
+	SRC_URI="http://www.panda3d.org/download/${P}/${P}.tar.gz"
+	S="${WORKDIR}"/${P}
+else
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/panda3d/panda3d.git"
+	EGIT_BRANCH="master"
+fi
+
+DESCRIPTION="A 3D game engine and framework for Python and C++."
+HOMEPAGE="http://www.panda3d.org"
+
+LICENSE="BSD"
+SLOT="0"
+
+IUSE="bullet +eigen ffmpeg fftw fmod jpeg opencv png +python +openal +ssl tiff truetype zlib"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+DEPEND="
+	bullet? ( sci-physics/bullet )
+	eigen? ( dev-cpp/eigen:3 )
+	ffmpeg? ( media-video/ffmpeg )
+	fmod? ( media-libs/fmod:= )
+	jpeg? ( virtual/jpeg:= )
+	opencv? ( media-libs/opencv )
+	png? ( media-libs/libpng:= )
+	${PYTHON_DEPS}
+	openal? ( media-libs/openal )
+	ssl? ( dev-libs/openssl:= )
+	tiff? ( media-libs/tiff:= )
+	truetype? ( media-libs/freetype )
+	zlib? ( sys-libs/zlib )
+	virtual/opengl"
+RDEPEND="${DEPEND}"
+# Note: libav may or may not work instead of ffmpeg, but in any case is not recommended by upstream and untested. If you try and it works please add it as an option, but include a warning in postinst.
+
+src_compile() {
+	multijob_init # mandatory according to docs
```

Comment:
i thought it's necessary per https://devmanual.gentoo.org/eclass-reference/multiprocessing.eclass/index.html